packages/frontity-org-theme/src/processors/scrolling-section.ts

```diff
@@ -0,0 +1,92 @@
+import { Processor } from "@frontity/html2react/types";
+import { css } from "frontity";
+import React from "react";
+
+import FrontityOrg from "../../types";
+import { addAlpha } from "../utils";
+
+export const scrollingSection: Processor<
+  React.HTMLProps<HTMLElement>,
+  FrontityOrg
+> = {
+  name: "scrolling-section",
+  test: ({ node }) =>
+    node.type === "element" &&
+    node.props?.className?.split(" ").includes("carrousel-bar"),
+  processor: ({ node, state }) => {
+    if (node.type !== "element") return node;
+
+    const thumbColor = state.theme.colors.frontity;
+    const trackColor = addAlpha(state.theme.colors.primary, 0.08);
+
+    node.props.css = css`
+      ${node.props.css};
+
+      /* This is the default breakpoint for Gutenberg's columns */
+      @media only screen and (max-width: 599px) {
+        overflow-x: auto;
+
+        /* Display the content in a row */
+        &.wp-block-group > .wp-block-group__inner-container {
+          padding: 24px 16px;
+          display: flex;
+        }
+
+        /* Make columns block to display the content in a row */
+        .wp-block-columns {
+          flex-direction: row;
+          flex-wrap: nowrap;
```

Comment:
Just FYI: you can use `flex-flow: row nowrap;`. For me it helps me to not forget to specify both flex-direction and flex-wrap.